fe/fe-core/src/main/java/com/starrocks/catalog/TabletStatMgr.java

```diff
@@ -337,7 +337,9 @@ private String debugName() {
         private void sendTasks() {
             Map<ComputeNode, List<TabletInfo>> beToTabletInfos = new HashMap<>();
             for (Tablet tablet : tablets.values()) {
-                ComputeNode node = Utils.chooseNode((LakeTablet) tablet);
+                ComputeNode node = GlobalStateMgr.getCurrentState().getWarehouseMgr()
+                        .getComputeNodeAssignedToTablet(WarehouseManager.DEFAULT_WAREHOUSE_NAME, (LakeTablet) tablet);
```

Comment:
‌‌‌‌‌‌‌How can we ensure that we are using the correct Warehouse Name? For example, if this PR is merged into the Enterprise Edition and the expected warehouse in the Enterprise Edition code is non-default warehouse, an incorrect modification may occur due to code merging.

Reply:
‌In the enterprise version, will there still be an error if there is no node in the Default Warehouse?

Reply:
Yes, your concern is correct, there is a BUG here, but I still try my best to migrate the logic first. In fact, there are currently many locations that directly use default_warehouse. This PR is already too large and is not suitable for repairing this problem.